Makefile

```diff
@@ -13,6 +13,7 @@ install:
 install-tests:
 	pip install -U setuptools pip
 	python3 tests_common/copy_files_from_core.py
+	isort -rc tests_common/
```

Comment:
Why was this line added here?

Reply:
The import `from utils...` in dump_tool.py is changed to `from inmanta_tests.utils...` which is in the wrong position. Instead of finding the correct position 'by hand', I thought it would be easiest to just use isort on the copied files to make sure everything is in the right place.